ibattery.py

```diff
@@ -0,0 +1,237 @@
+from libqtile.widget import base
```

Comment:
Include a docstring to explain to users how they can use your widget i.e.
``` 
from ibattery import Battery
```
etc.